### PR TITLE
EP-421/Android: CTA Clicked (project)

### PR DESCRIPTION
**Pull request by hadia**
# 📲 What

send CTA Clicked (project)User taps on a project from the results on the search page.

# 🤔 Why

1. Event Name -> CTA Clicked
2.  context_cta ->  ​project
3.  context_page ->  ​search
4.  context_location ->  ​search_results
5.  context_type ->  ​results

# 🛠 How

image
image

# 👀 See

Screen Shot 2021-04-21 at 3 49 15 PM

# 📋 QA

the open project as a search result

# Story 📖

`https://kickstarter.atlassian.net/browse/EP-421`

### CODE REVIEW

File: app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt [hunk 1]
```diff
@@ -749,6 +751,27 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track(CTA_CLICKED.eventName, props)
     }
 
+    /**
+     * Tracks a discover project clicks on the search result.
```
Comment (leighdouglas) [inline app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt:755]: Please add parameter documentation to this

Comment (hadia) [reply app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt]: Done ✅

File: app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt [hunk 2]
```diff
@@ -749,6 +751,27 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
...
+     */
+    fun trackDiscoverSearchResultProjectCATClicked(
```
Comment (leighdouglas) [inline app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt:761]: please correct the spelling of this method -> `trackDiscoverSearchResultProjectCTAClicked`

**Review by leighdouglas — APPROVED** [review]
Please address the comments and it's good to go! 🎉

**Review by Arkariang — APPROVED** [review]
lgtm

**Pull request merged**